src/main/java/com/stuypulse/robot/constants/Settings.java

```diff
@@ -45,13 +46,17 @@ static void reportWarning(String warning) {
 
     public interface Climber {
         boolean ENABLE_TILT = true;
-
-        SmartBoolean ENABLE_SWITCHES = new SmartBoolean("Climber/Enable Switches", false);
-
+        
         SmartNumber CLIMBER_DEFAULT_SPEED = new SmartNumber("Climber/Default Speed", 1.0);
         SmartNumber CLIMBER_SLOW_SPEED = new SmartNumber("Climber/Slow Speed", 0.2);
 
         SmartNumber CLIMBER_DELAY = new SmartNumber("Climber/Delay", 0.1);
+
+        SmartNumber CLIMBER_HEIGHT_LIMIT = new SmartNumber("Climber/Height Limit", -1);
+
+        double CLIMBER_GEAR_RATIO = 1.0 / 36;
+        double CLIMBER_MOTOR_DIAMETER = 2; //inches
```

Comment:
Are you sure this is the motor diameter? If this a number from engineering , it might be diameter of something the climber motor is 1:36 geared to

Reply:
yes, I dmed Kano

Reply:
Hm.. Idk what the mechanism looks like but the name MOTOR_DIAMETER makes me nervous

(also for clarity i wasn't talking about if the number was correct, I was talking about if this is the right context for using a motor diameter)